rose/web/game.js

```diff
@@ -1,43 +1,39 @@
-var ROSE = (function() {
-    "use strict";
```

Comment:
Since it was omitted as part of the patch, just to make sure:
Does the UI work as expected and does the game behave as normal?
I haven't tested it

Reply:
:fire: yes, works for me :-)

[Screencast from 2023-09-06 17-24-09.webm](https://github.com/RedHat-Israel/ROSE/assets/2181522/fd1691a2-0462-4e18-9887-c0a386ab41e7)

Reply:
Modules always run in strict mode

As well, all code in class bodies runs in strict mode